layouts/partials/header.html

```diff
@@ -170,6 +170,17 @@
                             </a>
                           </li>
 
+                          <li class="submenu-item p-3">
```

Comment:
We have 2 times forum here 
![image](https://github.com/simple-login/website/assets/3167780/d7c8a75c-6792-4392-a2de-bd0205306311)

I think as Github community is currently bigger, we don't add the link to forum.simplelogin.io here